app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
+    if user&.authenticate(params[:password])
```

Comment:
I learned two new things with this line, the `&` and `authenticate`:
- `&` will check if there is a nil value before running the code, which was useful in this situation if a user could not be found with the given email
- `authenticate` is executed on a model (generally a User) and takes a given password as an argument and attempts to match the user with the password